storage/client/queue/src/main/java/com/azure/storage/queue/QueueAsyncClient.java

```diff
@@ -108,6 +108,9 @@ public URL getQueueUrl() {
      *
      * {@codesnippet com.azure.storage.queue.queueAsyncClient.create}
      *
+     * <p>For more information, see the
+     * <a href="https://docs.microsoft.com/en-us/rest/api/storageservices/create-queue4">Azure Docs</a>.</p>
```

Comment:
Instead of generic "Azure Docs", it's nice to see where this links to like: "Azure Docs: Create queue"

Reply:
Yup, I too think something like this "Azure Docs: Create queue" could be useful!

Reply:
+1, will this mean, that we will need to update the azure docs format in blob api too, to ensure consistency?